.github/workflows/build-git-annex-macos.yaml

```diff
@@ -0,0 +1,206 @@
+name: Build git-annex snapshot for macOS
+
+on:
+  # Trigger the workflow on pull request,
+  pull_request:
+    paths:
+      - '.github/workflows/build-git-annex-macos.yaml'
+  schedule:
+    - cron: '30 01 * * *'
+
+env:
+  LANG: C
+
+jobs:
+  build-package:
+    runs-on: macos-latest
+    strategy:
+      # Do not fail fast so we could upload full build log and possible source
+      # pkgs even if binary package building fails
+      fail-fast: false
+    steps:
+    - name: Set up system
+      shell: bash
+      run: |
+        brew install libmagic pkg-config
+
+    - name: Checkout this repository
+      uses: actions/checkout@v1
+
+    - name: Building it all
+      run: |
+        scripts/ci/build_git_annex_macos
+
+    - name: Upload packages
+      uses: actions/upload-artifact@v2-preview
+      with:
+          name: git-annex-macos-dmg
+          path: git-annex/tmp/git-annex.dmg
+
+  test-annex:
+    runs-on: macos-latest
+    needs: build-package
+    strategy:
+      matrix:
+        flavor: [normal, crippled-tmp, crippled-home]
```

Comment:
just remove `crippled-home` from this list with a comment and url to git annex issue to readd it whenever joey fixes it.  I need to do that for the NFS testing I had added recently but had to disable: https://github.com/datalad/datalad-extensions/blob/master/.github/workflows/build-git-annex-debianstandalone.yaml#L60   That issue is fixed in annex but build is failing -- yet to finish troubleshooting in https://github.com/datalad/datalad-extensions/pull/32

Reply:
> with a comment and url to git annex issue

URL to what issue?  #30?  This PR?  Something else?

Reply:
git annex issue: https://git-annex.branchable.com/bugs/gets_stuck_in_Remote_Tests___40____63____41___while_running_on_OSX_with_HOME_on_crippled_FS/?updated   since that is the one, when fixed, would let us add `crippled-home` back to the matrix